rpcserver.go

```diff
@@ -109,6 +110,10 @@ const (
 	// proofTypeReceive is an alias for the proof type used for receiving
 	// assets.
 	proofTypeReceive = tapdevrpc.ProofTransferType_PROOF_TRANSFER_TYPE_RECEIVE
+
+	// MetadataPrecisionKey is the JSON key used in the metadata field of a
+	// minted asset to express the precision of the minted asset.
+	MetadataPrecisionKey = "tap_asset_precision"
```

Comment:
Could drop the prefix? Given it's JSON, which is already effectively namespaced (lives in the meta field), and we can save some here re space by doing so.

Reply:
ah yeah, def can drop
just used this to minimize conflict with some `precision` the user may have defined

but sgtm can just do `precision` here